src/main/java/com/google/graphgeckos/dashboard/storage/BuildbotNotFoundException.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.storage;
+
+/**
+ * This exception signals that a request has been made for a "index" entry to the database
+ * but there is no entry with that key.
+ */
+public class BuildbotNotFoundException extends IllegalArgumentException {
```

Comment:
What does this argument mean? Please rename it so that it reflects its meaning

Reply:
Name taken directly from the Oracle Reference JavaDoc for the constructor that it uses. https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalArgumentException.html

Reply:
The constructor will still work if you rename this argument to ```message```, for example